tetrateci/test_1.8.sh

```diff
@@ -27,28 +32,28 @@ go test -count=1 -tags=integ ./tests/integration/security/mtlsk8sca/...  -p 1 -t
 go test -count=1 -tags=integ ./tests/integration/security/sds_egress/...  -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/sds_tls_origination/...  -p 1 -test.v
 go test -count=1 -tags=integ ./tests/integration/security/webhook/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/sds_ingress/.  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_gateway/.  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/sds_ingress_k8sca/.  -p 1 -test.v
 
 if [[ ${CLUSTER} == "eks" ]]; then
-  git apply tetrateci/eks-ingress.1.8.patch
-  go test -count=1 -tags=integ -timeout 30m ./tests/integration/telemetry/stats/... -run='TestStatsFilter|TestSetup|TestIstioctlMetrics|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  -p 1 -test.v
   go test -count=1 -tags=integ ./tests/integration/pilot/endpointslice/. -istio.test.skipVM true  -p 1 -test.v
```

Comment:
Do we know why this wouldn't work in GKE/AKS?

Reply:
It wasn't here earlier and haven't got the chance to test it out on the other 2, so I would refrain from putting it there till I confirm that they run.